hook.go

```diff
@@ -65,18 +65,17 @@ func (h hook) Fire(entry *logrus.Entry) error {
 		err error
 	)
 
+	msg := entry.Message
+	if entry.Data["msg"] != nil {
+		msg = fmt.Sprintf("%s - %v", msg, entry.Data["msg"])
+	}
+
 	if entry.Data["error"] != nil {
 		err = entry.Data["error"].(error)
 
 		errorTxt := new(bytes.Buffer)
 		errorTxt.WriteString(err.Error())
```

Comment:
Here the error will be `<nil> - something`, don't you want to hide completely the `<nil>` part please?

Reply:
No

Reply:
https://github.com/Scalingo/logrus-rollbar/pull/9/files#diff-6824f41d082d8d1e355aae9838a40779R73

Reply:
my bad, I misread